tests/functional/object/mcg/test_mcg_quota.py

```
@@ -273,3 +273,253 @@ def test_noobaa_space_available_using_cli(
+                    break
+        except TimeoutExpiredError:
+            assert False, (
+                f"Quota status did not reach {QuotaStatus.EXCEEDING} "
+                f"within 360s, last observed: {last_seen}"
+            )
+
+        # 7. Increase max-size to just above current usage, verify APPROACHING_QUOTA
+        approaching_size_gi = 3
+        logger.info(
+            f"Increasing max-size quota to {approaching_size_gi}Gi "
+            "(just above current usage)"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)